rollup.config.js

```diff
@@ -1,14 +1,34 @@
 import nodeResolve from "@rollup/plugin-node-resolve";
-import typescript from "@rollup/plugin-typescript";
+import babel from "@rollup/plugin-babel";
 
-export default {
-  input: "./tests/index.ts",
-  output: {
-    file: "./tests/bundle.js",
-    format: "cjs",
+const zenBabelPlugins = require("zen-observable/scripts/babel-plugins.js");
```

Comment:
Not sure it’s wise to depend on something like an internal zen-observable script, can it just be inlined here? Although zen-observable doesn’t change too often.

Reply:
Good point! I will pin the exact version of `zen-observable` to make sure it doesn't change on us.

Reply:
https://github.com/apollographql/zen-observable-ts/pull/105/commits/ee819008533d817aa0ed61c40824f54f1922d150